CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
+## 3.9.0-RELEASE
+* `sendPrecompiledLetter` added to NotificationClient
+  - The client can now send PDF files which conform to the Notify printing template
+  - Send a Java File object or a base64 encoded string
+  - 'reference' must be provided to identify the document
```

Comment:
Maybe we should mention that since this is a new feature that it will only be available by invitation.

Reply:
As discussed I'll leave that out as its more with regard to the API permission etc